File: 657. Robot Return to Origin.py
#* 657. Robot Return to Origin
moves = "UD"
def judge_circle(moves):
    return moves.count('U') == moves.count('D') and moves.count('L') == moves.count('R')

# Example usage
moves = "UDLR"
print(judge_circle(moves))  # Output: True

File: 657. Robot Return to Origin.js
// 657. Robot Return to Origin
moves = "UD"
Array.prototype.count = function(val) {
  return this.filter(x => x === val).length;
};

function judgeCircle(moves) {
  const moveArray = moves.split('');
  return moveArray.count('U') === moveArray.count('D') && moveArray.count('L') === moveArray.count('R');
}

// Example usage
var moves = "UDLR";
console.log(judgeCircle(moves));  // Output: true
